common/locales/en/defaultTasks.json

```diff
@@ -11,6 +11,20 @@
   "defaultTodoNotes": "You can either complete this To-Do, edit it, or remove it.",
 
   "defaultTodo1Text": "Join Habitica (Check me off!)",
+  "defaultTodo1Notes": "You can either complete this To-Do, edit it, or remove it.",
+
+  "defaultTodo2Text": "Learn About Habitica",
+  "defaultTodo2Notes": "Overview of New Users:\n\n💻 only: ☰(Menu - Top Bar) > Help\n\nFAQ:\n\n📱: ☰ > Help & FAQ\n\n💻 : ☰ > Help > FAQ",
```

Comment:
Personally, I don't like this. Looks messy.

![screen shot 2016-09-12 at 8 09 44 am](https://cloud.githubusercontent.com/assets/2916945/18436941/64805720-78c0-11e6-8087-4acc49f72c4c.png)